pkg/jx/cmd/delete_vault.go

```diff
@@ -162,14 +166,19 @@ func (o *DeleteVaultOptions) removeGCPResources(vaultName string) error {
 		o.GKEZone = zone
 	}
 
-	sa := gke.VaultServiceAccountName(vaultName)
+	clusterName, err := gke.ShortClusterName(o.Kube())
+	if err != nil {
+		return err
+	}
+
+	sa := gke.VaultServiceAccountName(vaultName, clusterName)
```

Comment:
should this be tied to GKE here? others that use GKE have GKE in the function name so it is obvious